presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -178,15 +178,15 @@ private static AWSCredentialsProvider getAwsCredentialsProvider(GlueHiveMetastor
     {
         if (config.getAwsAccessKey().isPresent() && config.getAwsSecretKey().isPresent()) {
             return new AWSStaticCredentialsProvider(
-                new BasicAWSCredentials(config.getAwsAccessKey().get(), config.getAwsSecretKey().get()));
+                    new BasicAWSCredentials(config.getAwsAccessKey().get(), config.getAwsSecretKey().get()));
```

Comment:
Please also fix indentation in `presto-hive/src/main/java/io/prestosql/plugin/hive/ConnectorObjectNameGeneratorModule.java`, `presto-hive/src/main/java/io/prestosql/plugin/hive/HivePartitionManager.java` and  `presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java`